packages/core/admin/ee/admin/src/constants.ts

```diff
@@ -21,9 +21,23 @@ export const ADMIN_PERMISSIONS_EE = {
       read: [{ action: 'admin::provider-login.read', subject: null }],
       update: [{ action: 'admin::provider-login.update', subject: null }],
     },
+    releases: {
+      read: [
+        {
+          action: 'plugin::content-releases.settings.read',
+          subject: null,
+        },
+      ],
+      update: [
+        {
+          action: 'plugin::content-releases.settings.update',
+          subject: null,
+        },
+      ],
+    },
   },
```

Comment:
just curious, why do we need this set of constant permissions in the admin package ? 
PRobably same as for review workflows, is it not possible to define this permissions only inside the content-releases packaeg?

Reply:
nvm, I just realized this is necessary to make use of the useTypedSelector api